app/src/sharedTest/java/org/oppia/android/app/player/exploration/BUILD.bazel

```diff
@@ -0,0 +1,102 @@
+"""
+Tests for notices shown in the app.
```

Comment:
I thought I left a comment about this, but can't seem to find it. This is out of date (probably from a copy & paste?)--please update.

Reply:
Updated, Thanks for correcting me